ui/app/components/clients/horizontal-bar-chart.js

```diff
@@ -213,7 +213,7 @@ export default class HorizontalBarChart extends Component {
     // add client count total values to the right
     chartSvg
       .append('g')
-      .attr('transform', `translate(${CHART_MARGIN.left}, ${TRANSLATE.down})`)
+      .attr('transform', `translate(${TRANSLATE.left}, ${TRANSLATE.down})`)
```

Comment:
This adds a space next to the number on the horizontal chart. 
![image](https://user-images.githubusercontent.com/6618863/155574244-12d6e2f4-4ee2-4a09-9ab9-cc9f8191336d.png)